kentik_api_library/kentik_api/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .kentik_api import KentikAPI
 from .public import *
+from requests.packages.urllib3.util.retry import Retry
```

Comment:
Note, the purpose of importing the `Retry` class here is to make it simpler for end-users to import it when they want to customize the strategy. The can  use `from kentik_api import Retry` as opposed to having to use the long(ish) original path.